dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -247,145 +248,29 @@ def generate_workshops_survey_summary(workshop)
     workshop_summary
   end
 
-  # Below functions generate fake data.
-  def get_pre_workshop_surveys(workshop)
-    rand(10..20).times.map do |_|
-      {
-        general: {
-          how_excited: rand(3..5),
-          lunch_aspirations: %w(Tacos Burritos Pizza Sandwiches).sample
-        }
-      }
-    end
-  end
-
   def get_surveys_for_workshops(workshop)
+    pre_workshop_submissions = Pd::WorkshopDailySurvey.where(pd_workshop: workshop, day: 0)
+
     {
-      'Day 1' => rand(10..20).times.map do |_|
-        {
-          general: {
-            how_was_intro: rand(3..5),
-            bakers_speech_feedback: %w(Cool Awesome Funny Weird).sample
-          },
-          facilitator: {
-            rate_your_facilitator: {
-              500 => rand(1..5),
-              501 => rand(4..5)
-            },
-            describe_your_facilitator: {
-              500 => %w(Helpful Effective Smart Funny Engaging Boring).sample,
-              501 => %w(Amazing Super Brilliant Perfect).sample
-            }
-          }
-        }
-      end,
-      'Day 2' => rand(10..20).times.map do |_|
-        {
-          general: {
-            how_was_day_2_activity: rand(3..5),
-            how_was_day_2_food: rand(2..5),
-            cats_or_dogs: %w(Cats Cats! Dogs Puppies! Lizards).sample
-          }
-        }
-      end,
-      'Day 3' => rand(10..20).times.map do |_|
-        {
-          general: {
-            how_was_day_3_activity: rand(3..5),
-            how_were_animals: rand(4..5),
-            favorite_sport: %w(Football Baseball Basketball Soccer Hockey Judo).sample
-          }
-        }
-      end,
-      'Day 4' => rand(10..20).times.map do |_|
-        {
-          general: {
-            how_was_day_4_activity: rand(3..5),
-            how_was_meeting_lebron: rand(1..5),
-            favorite_tv_show: %w(Westworld Brooklyn\ 99 West\ Wing The\ Wire Breaking\ Bad).sample
-          }
-        }
-      end,
-      'Day 5' => rand(10..20).times.map do |_|
-        {
-          general: {
-            how_was_day_5_activity: rand(4..5),
-            how_was_meeting_andy_sandberg: rand(4..5),
-            how_got_home: %w(Walk Rideshare Bus Car Train).sample,
-            how_do_you_feel: %w(Good Great Awesome Amazing Excellent Fantabulous).sample
-          }
-        }
-      end,
+      'Pre Workshop' => {
+        general: pre_workshop_submissions.map(&:form_data_hash)
+      }
     }
   end
 
-  def get_post_workshop_surveys(workshops)
-    rand(10..20).times.map do |_|
-      {
-        general: {
-          overall: rand(4..5),
-          how_prepared: rand(4..5),
-          any_feedback: %W(It\ was\ great! I'm\ psyched! More\ cats\ next\ time).sample,
-          last_words: %W(Hasta\ la\ vista Peace Sayonara Nope).sample
-        }
-      }
+  def get_questions_for_forms(workshop)
+    pre_workshop_general = Pd::SurveyQuestion.find_by(form_id: CDO.jotform_forms['local']['day_0'])
```

Comment:
hardcode day_0? What about days 1-5?

Reply:
I'm hardcoding pre_workshop with this. Next change will start pulling in Days 1-5.